lib/scrabble.rb

```diff
@@ -15,4 +22,26 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def greater_than_7_bonus(word)
+    score = 0
+    word_length = word.length
+    if word_length >= 7
+      10 + score
+    else
+      score
+    end
+  end
+
+
+  def score_with_mulitpliers(word, letter_multipliers, multiplier = 1)
+    #require 'pry'; binding.pry
+    letter_values = point_values
+    bonus = greater_than_7_bonus(word)
+    word_value = word.split("").each_with_index.map do |letter, index|
+      letter_values[letter.upcase] * letter_multipliers[index]
+    end
+    word_value.reduce(:+) * multiplier + (bonus * multiplier)
```

Comment:
If you're using Ruby 2.4, use `#sum` instead of `#reduce`; it's a bit more readable